src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
```

Comment:
![image](https://user-images.githubusercontent.com/50755505/215785503-ccb78c1e-61cb-4d78-a5cc-913e9b7fcd7d.png)

It ruins you lots of tests.
Checkout mockup one more time and change color.